testfixtures_test.go

```diff
@@ -113,6 +113,40 @@ func TestQuoteKeyword(t *testing.T) {
 	}
 }
 
+func TestCheckDatabaseName(t *testing.T) {
+	// testdata
+	tests := []struct {
+		name       string
+		isGoodName bool
+	}{
+		{"db_test", true},
+		{"dbTEST", true},
+		{"testdb", true},
+		{"production", false},
+		{"productionTestCopy", true},
+		{"t_e_s_t", false},
+		{"ТESТ", false}, // cyrillic T
+	}
+
+	var (
+		mockedHelper *MockHelper
+		c            *Context
+		err          error
+	)
```

Comment:
Remove these three variables from here, and declare them inside the range loop (with `:=`)

Reply:
Do you think this is a bad practice? I don't. I think the scope of a variable should be the smaller as possible. (In Go, variables declared inside a loop block only exists inside that block)

Reply:
I still disagree. It's 100% guaranteed that the compiler will optimize it. There's no reason to make code fancier because "we can't hope the compiler will optimize it".

But in order to move this forward, I'll change it myself 🙂